File: 백준/Bronze/2292. 벌집/벌집.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        scanner.close();
        
        int count = 0;
        N = N - 1 > 0 ? N - 1 : 0;

        if(N == 0){
            System.out.println(1);
        } else {
            while (N > 0){
                N -= count * 6;
                count += 1;
            }
            System.out.println(count);
        }
    }
}

File: 백준/Bronze/2292. 벌집/벌집.cc
#include <stdio.h>

int main(){
    int N, count = 0;
    scanf("%d", &N);

    N = N - 1 > 0 ? N - 1 : 0;

    if(N == 0){
        printf("1\n");
    } else {
        while (N > 0){
            N -= count * 6;
            count += 1;
        }
        printf("%d\n", count);
    }

    return 0;
}
